docpipeline/generate.py

```diff
@@ -86,6 +87,14 @@ def process_blob(blob, credentials, devsite_template):
     blob.download_to_filename(tar_filename)
     log.info(f"Downloaded gs://{blob.bucket.name}/{blob.name} to {tar_filename}")
 
+    # Check to see if api directory exists in the tarball.
```

Comment:
Will you add a test for this? We probably want another test case that does one or two of the same checks as `test_generate`. Can make a new `@pytest.fixture` that copies `testdata` into `tmpdir/api` then returns `tmpdir`.

Reply:
added! Please see.